webapp/src/pages/Settings.tsx

```diff
@@ -531,6 +532,15 @@ const Settings: React.FC<Props> = ({ onClose }) => {
                   ""
                 )}
               </React.Fragment>
+              <React.Fragment key="persistent_id">
+                <Typography variant="body2">persistent_id:</Typography>
+                {displayStringField(
+                  "persistent_id",
+                  resultingConfig.columns.persistent_id,
+                  "columns",
+                  ""
+                )}
+              </React.Fragment>
```

Comment:
The fragment is not necessary, you can remove it
```suggestion
              <Typography variant="body2">persistent_id:</Typography>
              {displayStringField(
                "persistent_id",
                resultingConfig.columns.persistent_id,
                "columns",
                ""
              )}
```